apps/server/src/modules/tldraw/domain/ws-shared-doc.do.ts

```diff
@@ -1,9 +1,9 @@
-import { Doc } from 'yjs';
+import { applyUpdate, Doc } from 'yjs';
 import WebSocket from 'ws';
-import { Awareness, encodeAwarenessUpdate } from 'y-protocols/awareness';
+import { applyAwarenessUpdate, Awareness, encodeAwarenessUpdate } from 'y-protocols/awareness';
 import { encoding } from 'lib0';
 import { TldrawWsService } from '@modules/tldraw/service';
```

Comment:
Please think about restructor of the code, to avoid adding the services to this do.

Reply:
done